etc/conf/atTelemetry.yaml

```diff
@@ -142,33 +143,13 @@ telemetry:
   value: declination
   collect_after_event: start_collection_event
 
- ROTPA:
-  device: ATPtg
-  Stype: Event
-  topic: currentTarget
-  value: rotPA
-  collect_after_event: start_collection_event
-
- RATEL:
-  device: ATPtg
-  Stype: Event
-  topic: currentTarget
-  value: ra
-  collect_after_event: start_collection_event
-
- DECTEL:
-  device: ATPtg
-  Stype: Event
-  topic: currentTarget
-  value: declination
-  collect_after_event: start_collection_event
-
- ROTPATEL:
-  device: ATPtg
-  Stype: Event
-  topic: currentTarget
-  value: rotPA
-  collect_after_event: start_collection_event
+# --- Will come from ATMCS rotator angle when available
+# ROTPA:
```

Comment:
There is also the rotator start/end values that come from the ATMCS

Reply:
Yep, but we should add that to another ticket.